performance/bulk-edit.php

```diff
@@ -2,12 +2,58 @@
 
 namespace Automattic\VIP\Performance;
 
-// Bulk edits of lots of posts can trigger slow term count queries for each post updated
-add_action( 'load-edit.php', function() {
-	if ( isset( $_REQUEST['bulk_edit'] ) ) {
-		wp_defer_term_counting( true );
-		add_action( 'shutdown', function() {
-			wp_defer_term_counting( false );
-		} );
-	}
-} );
+class Bulk_Edit {
```

Comment:
I like the idea of using namespaces for things like this. I think we're all used to using classes in place of namespaces from before PHP 5.3. This pattern is nice though: https://github.com/Automattic/vip-go-mu-plugins/blob/518d47280954dd0b7f394a1a45fce5b9fe2e7dff/lib/proxy/ip-forward.php#L3